docusaurus/docs/create-a-plugin/develop-a-plugin/build-a-panel-plugin.md

```diff
@@ -14,52 +14,45 @@ keywords:
   - dashboards
 ---
 
+import CreatePlugin from '@shared/create-plugin-frontend.md';
+import PluginAnatomy from '@shared/plugin-anatomy.md';
+
 ## Introduction
 
-Panels are the building blocks of Grafana. They allow you to visualize data in different ways. While Grafana has several types of panels already built-in, you can also build your own panel, to add support for other visualizations.
+Panels are one of the fundamental building blocks of Grafana, which allow you to visualize data in different ways. Grafana has several types of panels already built-in, and many more available in the [Plugin Catalog](https://grafana.com/grafana/plugins/).
 
-For more information about panels, refer to the documentation on [Panels](https://grafana.com/docs/grafana/latest/panels/).
+To add support for other visualizations, you can create your own panel plugin. Panels are [ReactJS components](https://reactjs.org/docs/components-and-props.html and can be scaffolded with the `create-plugin` tool.
 
-{{% class "prerequisite-section" %}}
+For more information about panels, refer to the documentation on [Panels](https://grafana.com/docs/grafana/latest/panels/).
 
 ### Prerequisites
 
-- Grafana >=7.0
+- Grafana >=9.0
 - [LTS](https://nodejs.dev/en/about/releases/) version of Node.js
-- yarn
-  {{% /class %}}
-
-## Set up your environment
-
-{{< docs/shared lookup="tutorials/set-up-environment.md" source="grafana" version="latest" >}}
 
 ## Create a new plugin
 
-{{< docs/shared lookup="tutorials/create-plugin.md" source="grafana" version="latest" >}}
+<CreatePlugin pluginType="panel" />
 
 ## Anatomy of a plugin
 
-{{< docs/shared lookup="tutorials/plugin-anatomy.md" source="grafana" version="latest" >}}
+<PluginAnatomy />
 
 ## Panel plugins
 
-Since Grafana 6.x, panels are [ReactJS components](https://reactjs.org/docs/components-and-props.html).
-
-Prior to Grafana 6.0, plugins were written in [AngularJS](https://angular.io/). Even though we still support plugins written in AngularJS, we highly recommend that you write new plugins using ReactJS.
-
 ### Panel properties
 
-The [PanelProps](https://github.com/grafana/grafana/blob/747b546c260f9a448e2cb56319f796d0301f4bb9/packages/grafana-data/src/types/panel.ts#L27-L40) interface exposes runtime information about the panel, such as panel dimensions, and the current time range.
+The [PanelProps](https://github.com/grafana/grafana/blob/57960148e47e4d82e899dbfa3cb9b2d474ad56dc/packages/grafana-data/src/types/panel.ts#L74-L122) interface exposes runtime information about the panel, such as panel dimensions, and the current time range.
 
-You can access the panel properties through `props`, as seen in your plugin.
+You can access the panel properties through the `props` argument, as seen in your plugin.
 
-**src/SimplePanel.tsx**
+**src/components/SimplePanel.tsx**
```

Comment:
```suggestion
**`src/components/SimplePanel.tsx`**
```

Reply:
Do we have any more examples of file names for code snippets? Not essential, but I'm not a fan of how this looks on the page.

Reply:
I've made the suggested change for now